schutzbot/deploy.sh

```diff
@@ -6,9 +6,9 @@ function greenprint {
 }
 
 greenprint "Building test container"
-sudo podman build --label="quay.expires-after=1w" --security-opt "label=disable" -t image-builder-"${QUAY_REPO_TAG}" -f distribution/Dockerfile-ubi .
+sudo podman build --label="quay.expires-after=1w" --security-opt "label=disable" -t image-builder-crc -f distribution/Dockerfile-ubi .
 
-greenprint "Pulling osbuld/postgres:13-alpine"
```

Comment:
:laughing:

Reply:
:flushed:

Reply:
shall we just leave it in?

Reply:
You fixed it:
https://github.com/osbuild/image-builder-crc/pull/1590/commits/07c33bf3559841b3f15363c586e2773658e643c9#diff-f2926af8708379b42eb5702886f3ecc2f8be33915d8cf8394b7fe00d6b1692b6R11

Reply:
ye but the git history is a bit messy, could rebase it

Reply:
but then again, who cares!